extras/rya.pcj.fluo/pcj.fluo.app/src/main/java/org/apache/rya/indexing/pcj/fluo/app/export/kafka/KafkaBindingSetExporter.java

```diff
@@ -36,10 +36,11 @@
  * Incrementally exports SPARQL query results to Kafka topics.
  */
 public class KafkaBindingSetExporter implements IncrementalBindingSetExporter {
+    
```

Comment:
Okay.  Well that's good.  Can't tell if you're being sarcastic or something.  No need to leave a comment if nothing needs to be changed.